### PR TITLE
Add code of conduct (Issue #114)

### LINKED ISSUE #114: Add Code of Conduct to this repository
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/leaflet-environmental-layers/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Create** the file named [CODE_OF_CONDUCT.md] in the `leaflet-environment-layers` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

The file needs to be created at the root of the repository and copy the code of conduct from https://github.com/publiclab/plots2/blob/master/CODE_OF_CONDUCT.md

into newly created file. 

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

### PR DESCRIPTION
**Pull request by nkleinbaer**
Fixes #114

Copied CODE_OF_CONDUCT.md from /publiclab/plots2.

### CODE REVIEW

**Comment by gauravano** [comment]
Just a suggestion: Including Fixes #[issue-number] in your PR tags the relevant issue and close the issue when PR gets merged, so it's a good practice to include it. Thanks!

**Comment by gauravano** [comment]
Merged :tada: :balloon:! Thanks @nkleinbaer for nice work!

**Comment by gauravano** [comment]
@jywarren you have given me privileges to push to master in this repo. Feel free to update privileges if it's a mistake. And, I merged this one as it's a straight forward, hope you don't mind. Thanks!

**Comment by nkleinbaer** [comment]
> Just a suggestion: Including Fixes #[issue-number] in your PR tags the relevant issue and close the issue when PR gets merged, so it's a good practice to include it. Thanks!
> 
> I am mentioning the issue for now.

Noted, I wasn't sure one the formatting. Thank you!

**Comment by gauravano** [comment]
No issue. We will also update our PR template to include this as a suggestion by default. Thanks!

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+This document was copied from https://raw.githubusercontent.com/publiclab/plots2/master/CODE_OF_CONDUCT.md on January 28, 2019. See that page if you would like to submit your concerns in a safe, completely anonymous way, and to learn more about this document.
```
Comment (gauravano) [inline CODE_OF_CONDUCT.md]: Oh! Just noted this change after the merge :disappointed:. I think we need to use original content of  https://github.com/publiclab/plots2/blob/master/CODE_OF_CONDUCT.md with change in date only. @nkleinbaer can you open a PR for it so as to change the link? Thanks!

**Pull request merged**